docs/business.yaml

```diff
@@ -3700,31 +3877,27 @@ components:
             streetAddress: 5-4761 Bay Street
             streetAddressAdditional: Building A
             addressCity: Victoria
-            addressRegion: 'BC for British Columbia '
+            addressRegion: 'BC for British Columbia'
             addressCountry: Canada
             postalCode: V8R 2P1
             deliveryInstructions: '"Our apartment is located at the back of the building."'
           deliveryAddress:
             streetAddress: 5-4761 Bay Street
             streetAddressAdditional: Building A
             addressCity: Victoria
-            addressRegion: 'BC for British Columbia '
+            addressRegion: 'BC for British Columbia'
             addressCountry: Canada
             postalCode: V8R 2P1
             deliveryInstructions: '"Our apartment is located at the back of the building."'
     party:
       type: object
       title: party
-      description: 'Represent a person and its role in relation to a business. '
+      description: 'Represent a person and its role in relation to a business.'
       properties:
         party:
           type: object
-          description: 'Represent a person and its role in relation to a business. '
+          description: 'Represent a person and its role in relation to a business.'
```

Comment:
Noticed that the taxId field has been removed from the party schema. Would it be a good idea to update the x-examples by removing the taxId information as well?